docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -18,11 +18,12 @@ description: "Important considerations for WebLogic domains in Kubernetes."
 
 Please be aware of the following important considerations for WebLogic domains running in Kubernetes:
 
-* _Domain Home Location:_ The WebLogic domain home location is determined by the domain resource `domainHome` if set; otherwise, a default location is determined by the `domainHomeInImage` setting. If a domain resource `domainHome` field is not set
-  and `domainHomeInImage` is `true` (the default), then the operator will
+* _Domain Home Location:_ The WebLogic domain home location is determined by the domain resource `domainHome`, if specified; otherwise, a default location is determined by the `domainHomeSourceType` setting. If the domain resource `domainHome` field is not specified
```

Comment:
This bullet becomes too crowded.  Using sub-bullets or a table may help.

Reply:
Done. Please review the change, and close this comment if your good with it.